config/src/test/java/com/redhat/lightblue/config/TestPropertyParser.java

```diff
@@ -22,13 +22,4 @@
 import com.redhat.lightblue.metadata.parser.PropertyParser;
 
 public class TestPropertyParser extends PropertyParser {
-    @Override
-    public Object parse(String name, MetadataParser p, Object node) {
-        return null;
-    }
-
-    @Override
-    public void convert(MetadataParser p, Object emptyNode, Object object) {
-
-    }
 }
```

Comment:
To test loading of a custom property parser.

On Thu, Mar 31, 2016 at 5:22 PM, Dennis Crissman <notifications@github.com>
wrote:

> In
> config/src/test/java/com/redhat/lightblue/config/TestPropertyParser.java
> <https://github.com/lightblue-platform/lightblue-core/pull/624#discussion_r58143786>
> :
>
> > @@ -22,13 +22,4 @@
> >  import com.redhat.lightblue.metadata.parser.PropertyParser;
> >
> >  public class TestPropertyParser extends PropertyParser {
> > -    @Override
> > -    public Object parse(String name, MetadataParser p, Object node) {
> > -        return null;
> > -    }
> > -
> > -    @Override
> > -    public void convert(MetadataParser p, Object emptyNode, Object object) {
> > -
> > -    }
> >  }
>
> Why keep this class around?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/624/files/ea3a82c505fb57a14479315fd5b5ae19d4fdd0db#r58143786>
>